src/main/java/racingcarwinner/RacingCarWinner.java

```diff
@@ -0,0 +1,114 @@
+package racingcarwinner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCarWinner {
+
+    private static final int ADVANCE_MIN_VALUE = 4;
+    private String participantName;
+    private int carMovingCount;
+    private List<Car> carList = new ArrayList<>();
+
+    public RacingCarWinner(String participantName, int carMovingCount) {
+        this.participantName = participantName;
+        this.carMovingCount = carMovingCount;
+    }
+
+    private void splitParticipantName() {
+        String[] participantNames = participantName.split(",");
+        for (int i = 0; i < participantNames.length; i++) {
+            carList.add(new Car(participantNames[i]));
+        }
+    }
+
+    private int judgmentGoStop() {
+        int randomNum = (int) (Math.random() * 10);
+        if (randomNum >= ADVANCE_MIN_VALUE) {
+            return 1;
+        }
+        return 0;
+    }
+
+    private List<Integer> racingCarMove() {
+        List<Integer> carMovement = new ArrayList<>();
+        for (int i = 0; i < carList.size(); i++) {
+            carMovement.add(i, judgmentGoStop());
+        }
+        return carMovement;
+    }
+
+    private String updateParticipantInformation() {
+        List<Integer> carMovement = racingCarMove();
+        int eachNumberOfMove = 0;
+        for (int i = 0; i < carList.size(); i++) {
+            eachNumberOfMove = carList.get(i).getPosition() + carMovement.get(i);
+            carList.get(i).setPosition(eachNumberOfMove);
+        }
+        return printGameStatus();
+    }
+
+    private String printGameStatus() {
+        String printingStatus = "";
+        for (int i = 0; i < carList.size(); i++) {
+            printingStatus += carList.get(i).getCarName() + " : " + printCarBar(i) + "\n";
+        }
+        printingStatus += "\n";
+        return printingStatus;
+    }
+
+    private String printCarBar(int participantTurn) {
+        int moveDistance = carList.get(participantTurn).getPosition();
+        String distanceBar = "";
+        for (int i = 0; i < moveDistance; i++) {
+            distanceBar += "-";
+        }
+        return distanceBar;
+    }
+
+    private String winner() {
+        int max = carList.get(0).getPosition();
+        String winnerParticipant = "";
+        List<String> winnerResult;
+        for (Car carInformation : carList) {
+            winnerResult = maxDistanceCar(carInformation, max, winnerParticipant);
+            max = Integer.parseInt(winnerResult.get(0));
+            winnerParticipant = winnerResult.get(1);
+        }
+        if (winnerParticipant.charAt(0) == ',') {
+            return winnerParticipant.substring(1, winnerParticipant.length());
```

Comment:
String의 substring 메서드에 파라미터를 하나만 넘기면 파라미터로 넘긴 값의 뒤에 부분을 모두 반환합니다.
[공식문서](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#substring(int))를 한번 읽어봐도 좋습니다!
```suggestion
            return winnerParticipant.substring(1);
```